app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileServiceInstanceRow', {
+    controller: [
+      '$filter',
+      '$routeParams',
+      '$scope',
+      'APIService',
+      'AuthorizationService',
+      'BindingService',
+      'Constants',
+      'DataService',
+      'ListRowUtils',
+      'ServiceInstancesService',
+      MobileServiceInstanceRow
+    ],
+    controllerAs: 'row',
```

Comment:
There is a default controllerAs of `$ctrl`

Reply:
this is copied from serviceInstanceRow